provider/resources.go

```diff
@@ -289,6 +294,7 @@ func Provider() tfbridge.ProviderInfo {
 			"alicloud_cs_kubernetes_node_pool":   {Tok: resource(csMod, "NodePool")},
 			"alicloud_cs_edge_kubernetes":        {Tok: resource(csMod, "EdgeKubernetes")},
 			"alicloud_cs_kubernetes_permissions": {Tok: resource(csMod, "KubernetesPermission")},
+			"alicloud_cr_ee_instance":            {Tok: resource(csMod, "RegistryEnterpriseInstance")},
```

Comment:
It looks like the module is wrong

```suggestion
			"alicloud_cr_ee_instance":            {Tok: resource(crMod, "RegistryEnterpriseInstance")},
```